Assignment2/JustAntoRS-AntonioRamirez.ttl

```diff
@@ -0,0 +1,15 @@
+@base <http://example.org/>.
+@prefix : <http://example.org/properties>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 :includes :Sensor029;
+         :includes :Computer101.
+
+:Sensor029 :hasMeasurement :Measurement8401.
+
+:Computer101 :hasOwner :User10A.
+
+:Measurement8401 :hasTemperature "29":^^xsd:integer;
```

Comment:
Please, parse it, and you will see that there is an error.

Reply:
Done, sorry for all the iterations on this :sweat_smile: